locales/es-419.json

```diff
@@ -1 +1,195 @@
+  },
+  "help": {
+    "desc_highlight": "Es normal que aparezcan algunos errores y funcionalidades que aún estén en desarrollo.",
+    "desc_para1": "¡Gracias por tu interés en probar Elk, nuestro cliente genérico en desarrollo para Mastodon!",
+    "desc_para2": "Estamos haciendo lo posible para ir mejorando constantemente.",
+    "desc_para4": "Elk es de código abierto. Si quieres probar para ayudar, opinar o contribuir,",
+    "desc_para5": "contáctanos a través de GitHub"
+  },
+  "list": {
+    "add_account": "Añadir cuenta a la lista",
+    "remove_account": "Quitar cuenta de la lista"
+  },
+  "menu": {
+    "block_domain": "Ocultar dominio {0}",
+    "delete_and_redraft": "Eliminar y volver a borrador",
+    "edit": "Actualizar",
+    "pin_on_profile": "Anclar en tu perfil",
+    "show_favourited_and_boosted_by": "Ver quien marcó como favorita y quien retooteó",
+    "show_reblogs": "Ver retoots de {0}",
+    "unblock_domain": "Ver dominio {0}",
+    "unmute_account": "Quitar silencio a {0}",
+    "unmute_conversation": "Quitar silencio de la publicación",
+    "unpin_on_profile": "Desanclar del perfil"
+  },
+  "nav": {
+    "back": "Atrás",
+    "blocked_domains": "Dominios ocultos",
+    "built_at": "Generado {0}",
+    "conversations": "Mensajes directos",
+    "favourites": "Favoritas",
+    "federated": "Historia federada",
+    "local": "Historia local",
+    "settings": "Preferencias",
+    "toggle_theme": "Cambiar tema de color",
+    "zen_mode": "Modo sin distracciones"
+  },
+  "notification": {
+    "followed_you": "te siguió",
+    "update_status": "actualizó su publicación"
+  },
+  "placeholder": {
+    "default_1": "¿En qué piensas?"
+  },
+  "search": {
+    "search_empty": "No se encontraron resultados para la búsqueda"
+  },
+  "settings": {
+    "about": {
+      "built_at": "Compilado el",
+      "sponsor_action": "Patrocina"
+    },
+    "account_settings": {
+      "description": "Actualiza los ajustes de tu cuenta en la interfaz de Mastodon.",
+      "label": "Configuración de cuenta"
+    },
+    "interface": {
+      "color_mode": "Temas de color",
+      "dark_mode": "Tema oscuro",
+      "default": " (predeterminado)",
+      "font_size": "Tamaño de fuente",
+      "light_mode": "Tema claro",
+      "system_mode": "Color del sistema"
+    },
+    "language": {
+      "display_language": "Idioma en pantalla",
+      "translations": {
+        "add": "Añadir",
+        "hide_specific": "Ocultar una traducción específica",
+        "remove": "Quitar"
+      }
+    },
+    "notifications": {
+      "notifications": {
+        "label": "Preferencias de notificaciones"
+      },
+      "push_notifications": {
+        "label": "Preferencias de notificaciones push"
+      },
+      "show_btn": "Ir a preferencias de notificaciones",
+      "under_construction": "En desarrollo"
+    },
+    "preferences": {
+      "enable_autoplay": "Habilitar reproducción automática",
```

Comment:
move `enable_autoplay` to base `es` file

Reply:
move also `hide_favorite_count`, `hide_follower_count` and `hide_reply_count` to base `es` file

Reply:
"Move" meaning cut and paste from `es-419.json` to `es`, or just copy? Sorry for the silly question

Reply:
cut and paste